homeassistant/components/auth/__init__.py

```diff
@@ -637,8 +643,9 @@ async def websocket_delete_all_refresh_tokens(
     else:
         connection.send_result(msg["id"], {})
 
-    # This will close the connection so we need to send the result first.
-    hass.loop.call_soon(hass.auth.async_remove_refresh_token, current_refresh_token)
+    if not limit_token_types or current_refresh_token.token_type == token_type:
```

Comment:
Looking at https://github.com/home-assistant/frontend/pull/19480 it might make sense to add an option to not delete the current token. I think the most common use case is that you want to log out everything but yourself.